src/db/auth_db/mod.rs

```diff
@@ -124,15 +140,3 @@ impl DbUrls {
         self.create_bounce.clone()
     }
 }
-
-impl From<UrlError> for AppError {
-    fn from(error: UrlError) -> AppError {
-        AppErrorKind::AuthDbError(format!("{}", error)).into()
-    }
-}
-
-impl From<RequestError> for AppError {
```

Comment:
I also discovered that a couple of these blanket conversions were causing the wrong error response. Here, any `reqwest::Error` would end up as an `AuthDbError`, then somewhere else there was a similar case of it (I forget where now).